gradle.properties

```diff
@@ -1 +1 @@
-appVersion=0.0.1
+appVersion=0.0.2
```

Comment:
is this an our app version I think this should be part of gradle build file??

Reply:
Yes, gradle build uses this file to create artifact version